aws/disk.go

```diff
@@ -43,9 +44,12 @@ func (d *Disk) CreatedAt() time.Time { return *d.Volume.CreateTime }
 // Meta returns the disk metadata.
 func (d *Disk) Meta() unused.Meta { return d.meta }
 
-// SizeGB returns the size of this AWS EC2 volume in GB.
+// SizeGB returns the size of this AWS EC2 volume in GiB.
 func (d *Disk) SizeGB() int { return int(*d.Volume.Size) }
 
+// SizeBytes returns the size of this AWS EC2 volume in bytes.
+func (d *Disk) SizeBytes() int { return int(*d.Volume.Size) * int(math.Pow(1024, 3)) }
```

Comment:
Agreed with @jjo. Also note that here you're doing float64 to int just to do int to float64 when you emit the metric, so that's two unnecessary conversions.

Reply:
> you're doing float64 to int just to do int to float64

No, I'm not. The Size I get straight from the object here is an int32. Which is too small to fit the values in bytes. Granted, I could cast it straight to float64 and skip one cast. I was trying to keep consistency to the other size method. But I can change it to float64.